File: icd0017-2018s/radio/app/src/main/java/com/example/radio2019/Domain/Artist.java
package com.example.radio2019.Domain;

import java.util.ArrayList;

public class Artist {
    public int id;
    public String artistName;
    public ArrayList<Song> songs;

    public Artist(String artistName){
        this(0, artistName);
    }

    public Artist(int id, String artistName, ArrayList<Song> songs){
        this.id = id;
        this.artistName = artistName;
        this.songs = songs;
    }

    public Artist(int id, String artistName){
        this.id = id;
        this.artistName = artistName;
    }
}

File: icd0017-2018s/Radio_iOS/Radio_iOS/Domain/Artist.swift
//
//  Artist.swift
//  Radio_iOS
//
//  Created by user154645 on 5/11/19.
//  Copyright © 2019 user154645. All rights reserved.
//

import Foundation

class Artist {
    var id: Int
    var artistName: String
    var songs: [Song]
    
    convenience init(artistName: String){
        self.init(id: 0, artistName: artistName)
    }
    
    init(id: Int, artistName: String, songs: [Song]){
        self.id = id
        self.artistName = artistName
        self.songs = songs
    }
    
    init(id: Int, artistName: String){
        self.id = id
        self.artistName = artistName
        self.songs = []
    }
}
